style/json-viewer.css

```diff
@@ -0,0 +1,72 @@
+.json-viewer {
```

Comment:
Are these styles coming from the lib too?

Reply:
Yes